WordPress/WordPressUITests/Utils/XCTest+Extensions.swift

```diff
@@ -59,156 +20,14 @@ extension XCUIElement {
-    }
-
-    public func logoutSelfHosted() {
-        let app = XCUIApplication()
-
-        let removeButton = app.tables.cells[ elementStringIDs.removeSiteButton ]
-        let mySitesTabButton = app.tabBars[ elementStringIDs.mainNavigationBar ].buttons[ elementStringIDs.mainNavigationMySitesButton ]
-        let siteNameField = app.tables.staticTexts[ WPUITestCredentials.selfHostedSiteAddress ]
-
-        // Tap the My Sites button twice to be sure that we're on the All Sites list
-        mySitesTabButton.tap()
-        mySitesTabButton.tap()
-
-        siteNameField.tap()
-        app.tables.element(boundBy: 0).swipeUp()
-
-        waitForElementToAppear(element: removeButton)
-        removeButton.tap()
-
-        if isIpad(app: app) {
-            app.alerts.buttons.element(boundBy: 1).tap()
-        } else {
-            app.sheets.buttons.element(boundBy: 0).tap()
-        }
-    }
-
-    public func createAccount(email: String, username: String, password: String, url: String? = nil) {
-        let app = XCUIApplication()
-        let createSiteFromLoginScreenButton = app.buttons[ elementStringIDs.createSiteButton ]
-        let emailField = app.textFields[ elementStringIDs.nuxEmailField ]
-        let usernameField = app.textFields[ elementStringIDs.nuxUsernameField ]
-        let passwordField = app.secureTextFields[ elementStringIDs.nuxPasswordField ]
-        let urlField = app.textFields[ elementStringIDs.nuxUrlField ]
-        let createAccountButton = app.buttons[ elementStringIDs.nuxCreateAccountButton ]
-
-        if createSiteFromLoginScreenButton.exists {
-            createSiteFromLoginScreenButton.tap()
-        }
-
-        waitForElementToAppear(element: emailField)
-
-        emailField.tap()
-        emailField.clearAndEnterText(text: email)
-        usernameField.tap()
-        usernameField.clearAndEnterText(text: username)
-        passwordField.tap()
-        passwordField.clearAndEnterText(text: password)
-
-        if url != nil {
-            urlField.tap()
-            urlField.clearAndEnterText(text: url!)
-        }
-
-        createAccountButton.tap()
-    }
-}
-
-extension XCUIElement {
-    /**
-     Removes any current text in the field before typing in the new value
-     - Parameter text: the text to enter into the field
-     */
-    func replaceText(text: String) {
-        guard let stringValue = self.value as? String else {
-            XCTFail("Tried to clear and enter text into a non string value")
-            return
-        }
-
-        self.tap()
-        let deleteString = stringValue.map { _ in "\u{8}" }.joined(separator: "")
-
-        self.typeText(deleteString)
-        self.typeText(text)
-    }
-}
-
 extension XCTest {
 
     func isIPhone() -> Bool {
         let app = XCUIApplication()
         return app.windows.element(boundBy: 0).horizontalSizeClass == .compact || app.windows.element(boundBy: 0).verticalSizeClass == .compact
```

Comment:
Just a reminder about @jkmassel  comment for this one as well

```suggestion
        return UIDevice.current.userInterfaceIdiom == .phone
```